src/browser/modules/EditorInfo/EditNodes.jsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * This module depicts the behaviour of the edit drawer that displays the node relationship
+ properties
```

Comment:
Every line of JSDoc in multiline form should start with '*'

Reply:
done

Reply:
> done

not done

Reply:
Missing changes. Added